<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventana</class>
 <widget class="QDialog" name="ventana">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1305</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="ContenidoTxt">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>291</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonRegresar">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>0</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Regresar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonCrear">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Crear Publicación</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonSelecionar">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>210</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Seleccionar imagen</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelimagen">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>321</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>20</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Mostrar</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>20</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tablepubli">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>90</y>
     <width>521</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="linepubli">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>220</y>
     <width>281</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Buttonpubli">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>220</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Ver </string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="PubliTxt">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>311</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelpubli">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>541</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="tablecomen">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>260</y>
     <width>521</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Comentar">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>390</y>
     <width>281</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonComen">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>390</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Comentar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonOrden">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>60</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Inorden</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonPreorden">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>60</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Preorden</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonPostorden">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>60</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Postorden</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLimite">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>60</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonTop">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>620</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Top 3 Fechas con más publicaciones </string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonGraficar">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>20</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Graficar</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelGrafico">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>430</y>
     <width>531</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonMostrarTopPublicaciones">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>650</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Top 3 publicaciones con más  comentarios</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
